src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -163,10 +172,76 @@ class PacketDataReporter : public ParserDelegate
     void OnOperationalIPAddress(const chip::Inet::IPAddress & addr);
 };
 
+const char * ToString(mdns::Minimal::QType qtype)
```

Comment:
It might make sense to move this into the minimal implementation files. @andreilitvin - are there QType / QName utils somewhere?

Reply:
Removed this code after switching to the correct mDNS port number